legal-api/flags.json

```diff
@@ -12,7 +12,8 @@
         },
         "db-versioning": {
             "legal-api": true,
-            "emailer": false,
+            "emailer": true,
+            "tracker": true,
```

Comment:
Since tracker and entity-emailer are in separated project and has their own config.py, I added the flag for tracker here.
Let me know if you think it's not necessary and can just use emailer's feature flag.

Reply:
I was thinking, if only emailer using it, then we can probably just use `emailer`, but... from the Tracker db, I found entries from other services.
![image](https://github.com/user-attachments/assets/2c1ad5a8-fc72-4b8d-b796-3dfe2019ba15)

Reply:
I think we can just use the `emailer` feature flag. Only the emailer directly references the tracker code (can think of the tracker component as part of the emailer). Those sources are back-end components that have published queue messages to the NATS emailer, which uses the `tracker.MessageProcessingService` to save to the tracker db.

Reply:
thank you Leo, gonna adjust it